schemas/system/domains.schema

```diff
@@ -0,0 +1,7 @@
+{
+  IdStatus: Enum('Preallocated', 'Initialization', 'Actual', 'Historical'),
+  IdKind: Enum('Category', 'File', 'Cache'),
```

Comment:
Why not just `Status` and `Kind`?

Reply:
Because this enums are applicable only for `Identificator`, so other `Category` can have its own `Status` and `Kind` enums

Reply:
It looks like we need no `File` value in `IdKind` because `Identifier.Category` may refer to category `File`.